docs/_future/request_time_off_tutorial.md

```diff
@@ -526,11 +526,11 @@ To learn more about actions, visit the documentation [here](/bolt-js/concepts#ac
 ---
 ### Views and modals {#views-and-modals}
 
-Additional interactivity can be added in to your next-generation Slack app through [Modals](/bolt-js/concepts#creating-modals) and [Views](/bolt-js/concepts#updating-pushing-views). Within the Request Time Off example, we can add an additional button to the time off request message a manager receives that opens a new modal.
+Additional interactivity can be added into your next-generation Slack app through [Modals](/bolt-js/concepts#creating-modals) and [Views](/bolt-js/concepts#updating-pushing-views). Within the Request Time Off example, we can add an additional button to the time off request message a manager receives that opens a new modal.
 
 Having additional interactivity such as a modal could create a pathway for more functionality in the Request Time Off flow, such as allowing a manager to send a message to the requester to get additional information or clarification before approving or denying the request officially.
 
-Let's dive in to see how we can add a new button that launches a simple modal with a corresponding view!
+Let's dive in and see how we can add a new button that launches a simple modal with a corresponding view!
```

Comment:
This change seems reasonable and means that the basic rule "don't write `in to`" can be applied. But, if someone feels strongly for `dive in to see`, I'm happy to drop.